### PR TITLE
Feature/latest api backends - closes #687

### LINKED ISSUE #687: Implement database query and view for showing latest API backends
As a part of the front-page re-design we need a view of the newest additions to API catalog. This requires both a query (publication and subscription) and a view.

* [x] Create publication
* [x] Create subscription 
* [x] Create temporary route (For testing purposes, until home page is redesigned) 
* [x] Create template
* [x] Use specific query that contains sorting by `created_at` field & limit amount of returned documents, for example to 6.
* [ ] Integrate `latestApiBackends` template to a redesigned home page.

### PR DESCRIPTION
**Pull request by frenchbread**
#687 is related.

### CODE REVIEW

**Comment by brylie** [comment]
@frenchbread status report?

**Comment by frenchbread** [comment]
@brylie Please see a status report in #687.

As mentioned there, the task is done but need to decide weather to wait until redesign of a home page is implemented (to be able to put `lastApiBackends` template there) or, for now, to keep the template on a separate route. - Second approach is currently implemented.

**Comment by brylie** [comment]
OK, lets go ahead and review/merge this. We can add an enhancement task to add the view to the frontpage, so that this code can be merged.

**Comment by jykae** [comment]
I'll help by checking this one.

**Comment by brylie** [comment]
@jykae thanks!

**Comment by jykae** [comment]
I added one comment using the online editor to clarify the database query. Otherwise this looks good and it worked. We have a task https://github.com/apinf/api-umbrella-dashboard/issues/730 created by @brylie for including the feed to the frontpage.

File: client/views/latest_api/latestApi.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Template.latestApiBackends.helpers({
+  'latestBackends': function () {
+
+    // Subscribe to a publication
+    Meteor.subscribe("latestApiBackends");
```
Comment (brylie) [inline client/views/latest_api/latestApi.js]: Subscriptions should happen in the `Template.created` callback, rather than within template helpers.

**Pull request merged**